dev-lang/zig-bin/zig-bin-0.9.1-r1.ebuild

```diff
@@ -17,7 +17,12 @@ KEYWORDS="-* ~amd64 ~arm ~arm64 ~x86"
 
 RDEPEND="!dev-lang/zig"
 
-PATCHES=( "${FILESDIR}/${P}-fix-detecting-abi.patch" )
+SRC_URI+=" https://codeberg.org/BratishkaErik/distfiles/media/branch/master/zig-0.9.1-fix-detecting-abi.patch"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-bad-hostname-segfault.patch"
```

Comment:
... does this do anything? It's a -bin? How does that work?

Reply:
> ... does this do anything? It's a -bin? How does that work?
> 
> Or are parts of Zig evaluated at runtime?

It is part of Zig standard library, which Zig provides in "/opt/zig-bin-0.9.1/lib/" to be more flexible for patching like this

Reply:
Very interesting! It might be worth putting a note in the ebuild for future folks as I expect many of us won't realise that.